executor/aggfuncs/builder.go

```diff
@@ -120,7 +120,11 @@ func buildGroupConcat(aggFuncDesc *aggregation.AggFuncDesc, ordinal int) AggFunc
 
 // buildCount builds the AggFunc implementation for function "BIT_OR".
 func buildBitOr(aggFuncDesc *aggregation.AggFuncDesc, ordinal int) AggFunc {
-	return nil
+	base := baseAggFunc{
+		args:    aggFuncDesc.Args,
+		ordinal: ordinal,
+	}
```

Comment:
need to handle the function which has the `distinct` property.

Reply:
why bit-or need to care `distinct` property?

Reply:
consider this query: `select bit_or(distinct a) from t;` we only calculate the distinct values of column `a` in this kind of query.

Reply:
It doesn't matter, because `bit_or(distinct a) = bit_or(a)`, `bit_and` same too, except `bit_xor`.

Reply:
ok

Reply:
please add a comment here to statement that function `bitor` no need to consider the `distinct` property

Reply:
done